rusty_results/prelude.py

```diff
@@ -87,7 +87,7 @@ def map(self, f: Callable[[T], U]) -> "Option[U]":
         ...
 
     @abstractmethod
-    def map_or(self, default: U, f: Callable[[T], U]) -> "Option[U]":
+    def map_or(self, default: U, f: Callable[[T], U]) -> "U":
```

Comment:
No need to enclose `U` with `"`:
just `... -> U`

Reply:
Done